src/scene/scene_phongmaterial.js

```diff
@@ -660,9 +660,11 @@ pc.extend(pc, function () {
 
             if (this.cubeMap) {
                 this.setParameter('texture_cubeMap', this.cubeMap);
+                this.setParameter('material_cubemapSize', this.cubeMap.width);

```

Comment:
Any reason this isn't called texture_cubemapSize? It's a texture property rather than a material property. It's internal so it's not a big deal...